extensions/vertx-core/runtime/src/test/java/io/quarkus/vertx/core/runtime/VertxCoreProducerTest.java

```diff
@@ -100,6 +100,7 @@ private VertxConfiguration createDefaultConfiguration() {
         vc.maxWorkerExecuteTime = Optional.of(Duration.ofSeconds(1));
         vc.internalBlockingPoolSize = 20;
         vc.useAsyncDNS = false;
+        vc.preferNativeTransport = false;
```

Comment:
No need to initialize here too, I suppose? :)

Reply:
This method creates a default configuration and sets all values. I figured best to keep it consistent.